package.json

```diff
@@ -50,6 +50,7 @@
     "@babel/core": "patch:@babel/core@npm%3A7.23.2#./.yarn/patches/@babel-core-npm-7.23.2-b93f586907.patch",
     "@esbuild-plugins/node-modules-polyfill@^0.2.2": "patch:@esbuild-plugins/node-modules-polyfill@npm%3A0.2.2#./.yarn/patches/@esbuild-plugins-node-modules-polyfill-npm-0.2.2-f612681798.patch",
     "@lavamoat/lavapack@^6.1.1": "patch:@lavamoat/lavapack@npm%3A6.1.1#./.yarn/patches/@lavamoat-lavapack-npm-6.1.1-b81af21193.patch",
+    "@metamask/utils": "9.0.0",
```

Comment:
This yarn resolutions version fix is necessary because the changes in this PR need to be released in phases.

- `snaps-sdk` and `snaps-utils` are blocking releases of core monorepo packages
- `snaps-controllers` can't be released until these core packages are released with a `@metamask/utils` version bump:
  - `approval-controller`, `base-controller`, `json-rpc-engine`, `json-rpc-middleware-stream`, `permission-controller`

Independently of this, for the remaining `snaps` packages to be released, the following packages also need to have their `@metamask/utils` versions bumped:
  - core: `controller-utils`, `eth-json-rpc-provider`
  - standalone: `browser-passworder`, `create-release-branch`, `eth-block-tracker`, `eth-json-rpc-middleware`, `eth-sig-util`, `key-tree`, `post-message-stream`, `providers`, `snaps-registry`

Any `snaps` package releases should be put on hold while this process is ongoing. 
  - As an exception, packages that do not use `@metamask/utils` anywhere in their dependency tree (directly or via other dependencies) should have no issue being released.